src/Sylius/Bundle/CoreBundle/Form/Type/Taxation/TaxCalculationStrategyChoiceType.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Form\Type\Taxation;
```

Comment:
Can we put it in the same namespace as other form types? I agree we need to separation in the core, but I'd like to do it in separate PR if we agree that's a good idea.